templates/taxi/driver_list.html

```diff
@@ -1,10 +1,13 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Driver List
-    </h1>
+  <h1>Driver List
+    <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
```

Comment:
Ensure that the 'Create' button correctly leads to the driver creation page, as required by the task: 'On the driver list page create a button that leads to the driver creation page.'